PyInstaller/utils/hooks/qt.py

```diff
@@ -197,7 +197,7 @@ def qt5_qml_data(directory):
     Return Qml library directory formatted for data.
     """
     qmldir = qt5_qml_dir()
-    return os.path.join(qmldir, directory), os.path.join('PyQt5', 'Qt', 'qml')
+    return os.path.join(qmldir, directory), 'qml{0}{1}'.format(os.sep, directory)
```

Comment:
Thanks for updating the pull-request so quickly. @springermac advised you to use `os.sep`. But the only correct way to join files-system paths in a platform independent way is using `os.path.join()` – like it was before.